tile-json/map.js

```diff
@@ -0,0 +1,162 @@
+import { Loader } from '@googlemaps/js-api-loader';
+import axios from 'axios';
+import { GOOGLE_MAPS_TOKEN } from '@env';
+
+/**
+ * This example shows how to use the GeoJSOn response from our Vector Tile Server with Google Maps
+ *
+ * To establish a connection with Chargetrip GraphQL API you need to have an API key.
+ * The key sin this example is are public ones and gives access only to a part of our extensive database.
+ * You need a registered `x-client-id` to access the full database.
+ * Read more about an authorisation in our documentation (https://docs.chargetrip.com/#authorisation).
+ */
+
+// Add your own API Key in your .env file
+const loader = new Loader({ apiKey: GOOGLE_MAPS_TOKEN });
+
+let map;
+let markers = [];
+
+async function displayMap() {
+  const google = await loader.load();
+
+  map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 3.5,
+    center: { lat: 52.367, lng: 4.8979755 },
+    disableDefaultUI: true,
+  });
+
+  map.addListener('zoom_changed', () => {
+    hideMarkers();
+  });
+
+  addPolygon(google);
+
+  var maptiles = new google.maps.ImageMapType({
+    getTileUrl: function(coord, zoom) {
+      axios
+        .get(
+          `https://api.chargetrip.io/station/${zoom}/${coord.x}/${coord.y}/tile.json?connectors[]=IEC_62196_T2&connectors[]=IEC_62196_T2_COMBO&connectors[]=TESLA_S&connectors[]=CHADEMO&powerGroups[]=fast&powerGroups[]=turbo`,
+          {
+            headers: {
+              // Replace this x-client-id with your own to get access to more station data
+              'x-client-id': '5ed1175bad06853b3aa1e492',
+            },
+          },
+        )
+        .then(response => {
+          if (response.data) {
+            for (let i = 0; i < response.data.features?.length; i++) {
+              const feature = response.data.features[i];
+              if (feature.properties?.count > 1) {
+                addClusterMarker(google, feature);
+              } else {
+                addStationMarker(google, response.data.features[i]);
+              }
+            }
+          }
+        });
+    },
+    tileSize: new google.maps.Size(256, 256),
```

Comment:
Do we need axios for a single request like this? Can't we just use a promise. We don't do that much error handling anyway.

Reply:
Also we usually move network requests to a client.js file. This way all parts are structured the same way.